include/nuttx/can/can.h

```diff
@@ -235,9 +270,14 @@
 #define CANIOC_BUSOFF_RECOVERY    _CANIOC(10)
 #define CANIOC_SET_NART           _CANIOC(11)
 #define CANIOC_SET_ABOM           _CANIOC(12)
+#define CANIOC_IFLUSH             _CANIOC(13)
+#define CANIOC_OFLUSH             _CANIOC(14)
+#define CANIOC_IOFLUSH            _CANIOC(15)
+#define CANIOC_FIONWRITE          _CANIOC(16)
```

Comment:
should we reuse FIONWRITE/FIONREAD

Reply:
@xiaoxiang781216. Yes, by all means - I just went with the suggestion from the original issue that was raised. Makes more sense as the functionality is the same. I'll do it tomorrow.